src/components/EditCoursePage/index.jsx

```diff
@@ -657,9 +659,11 @@ class EditCoursePage extends React.Component {
       enterprise_subscription_inclusion,
       organization_short_code_override,
       organization_logo_override_url,
+      watchers,
```

Comment:
nit: add a comment to explain why watchers field is needed here alongside watchers_list.